TodoLogic/src/components/TodoList.tsx

```diff
@@ -0,0 +1,46 @@
+import TodoDescription from "./TodoDescription";
+import "./App.css";
+import { v4 as uuidv4 } from "uuid";
+import { useState, useEffect } from "react";
+
+const TodoList = (prop: { value: string }): JSX.Element => {
+  const [submit, setSubmit] = useState<string[]>(() => {
```

Comment:
**This callback function to initialize state is copied from jinal's code.**

Reply:
At first time I put these all code in useffect but this gives me error so they call me try this